Michael-Masas/bootstrap.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
-sudo yum install -y yum-utils;
+sudo yum install -y yum-utils
```

Comment:
No need to inline `sudo` commands.
Make sure that the user has root privileges and add nice comment at the start of the script that saying that "this scripts need's to run as root"